README.md

```diff
@@ -1,5 +1,8 @@
 # hpc-cluster-symphony
-Repository for the HPC Cluster Symphony implementation files.
+
```

Comment:
This log4j notice needs to be removed since the required remediations have been implemented in our code.  There is no longer any need for the user to take any of the mitigation steps.